pravega-client-examples/README.md

```diff
@@ -140,3 +140,53 @@ To demonstrate the use of `StreamCut`s, we provide a CLI. To use it, please exec
 $ bin/streamCutsCli [-scope myScope] [-name myStream] [-uri tcp://127.0.0.1:9090]
 ```
 
+## `secure`
+This example includes two applications `SecureReader` and `SecureWriter`. These applications illustrate
+how to setup clients connecting to a security-enabled Pravega cluster. Specifically, they show how to:
+* Enable SSL/TLS (HTTPS) communications with a Pravega cluster for data-in-transit encryption and server authentication.
+* Pass credentials to a Pravega cluster for client authentication and authorization.
+
+### Execution
+
+First, ensure that the Pravega cluster that the applications are going to work with, has both TLS and
+Auth (authentication and authorization) enabled. See Pravega documentation for how to enable TLS and Auth.
```

Comment:
Adding a link to the point in the documentation to do this would be great.

Reply:
Unfortunately, the documentation [here](https://github.com/pravega/pravega/blob/master/documentation/src/docs/deployment/run-local.md#running-pravega-in-standalone-mode-with-ssltls-enabled) talks about TLS only, and not Auth. Pravega security documentation needs some updates and those updates are pending. I don't see any document that I can point to as of now.

Moreover, I also feel that we shouldn't link explicitly to a document in another repo from code in this repo. This is because the locations and organization of documents in another repo can change and keeping the link up to date in the code in a timely manner can be challenging. What do you say?